3.train_ebimage/1.random_forest/README.md

```diff
@@ -30,8 +30,8 @@ Using [Sklearn Random Forest Classifier](https://scikit-learn.org/stable/modules
 
 ### Results
 
-The balanced f1-score on the validation set is [0.328](scripts/html/random_forest.html).
-This is very low, but what is interesting is the [confusion matrix](scripts/html/random_forest.html).
+The balanced f1-score on the validation set is [0.328 ± 0.0030 with n=6](results/all_scores.xlsx).
```

Comment:
```suggestion
The balanced f1-score on the validation set is [0.328 ± 0.0030 with 6 model initializations](results/all_scores.xlsx).
```

this is great to have a range!! nice work. I change n=6 since n usually refers to sample size and (i think) we're referring to 6 model initializations instead.